congestion-control-charter.txt

```diff
@@ -1,85 +1,34 @@
-﻿CONGestion RESponse and Signaling Working Group Charter (CONGRESS)
-
-RFC 5033 describes a Best Current
-Practice to evaluate new congestion control algorithms as Experimental or
-Proposed Standard RFCs. TCP was the dominant
-consumer of this work, and proposals were typically discussed in research groups (for example, Internet Congestion Control Research Group - ICCRG).
-
-
-Since RFC 5033 was published, many conditions have changed. Congestion control algorithm proponents
-now often have the opportunity to test and deploy at scale without IETF review. The
-set of protocols using these algorithms has spread beyond TCP and SCTP to
-include DCCP, QUIC, and beyond. There is more interest in specialized use cases
-such as data centers and real-time protocols. Finally, the community has gained
-much more experience with indications of congestion beyond packet loss.
-
-The CONGestion RESponse and Signaling working group (CONGRESS) can review some
-of the impediments to early congestion control work occurring in the IETF, and
-can generalize transport in this area from TCP to all of the relevant transport
-protocols. The congestion control expertise in the working group would also
-make it a natural venue to take on other work related to indications of
-congestion, such as delay, and potentially queueing algorithms.

```

Comment:
Seems like we still need something like this paragraph.

Reply:
I think this goes to the point of https://github.com/martinduke/congestion-control-charter/pull/35#discussion_r1152729095, so let's either include or remove it based on the outcome there.